web/src/components/people/ApprovedResearcherView.module.css

```diff
@@ -0,0 +1,8 @@
+.button-container {
+  justify-self: end;
+  justify-items: end;
+  height: 4rem;
+  display: grid;
+  margin-block: 0.5rem;
```

Comment:
might want to just set margin-block to be margin-bottom since you're overwriting the top anyway